src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -33,11 +38,33 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+var kafkaPortDefaults []string = []string{"9092"}
+var proxyPortDefaults []string = []string{"7072"}
+var registryPortDefaults []string = []string{"8081"}
+var adminPortDefaults []string = []string{"9644"}
+var brokerRPCPortDefaults []string = []string{"33145"}
+
```

Comment:
I was kind of aiming to reduce magic literals on this. But I am down to do whatever works. What do you think?

Reply:
I mean using the `config` defaults, something like:

```
	command.Flags().StringSliceVar(
		&kafkaPorts,
		"kafka-ports",
		[]string{strconv.Itoa(config.DefaultKafkaPort)},
		"List of ports that you want to use. Ports must be surely free otherwise containers won't start",
	)
```

This is to only have _one_ place to store the default ports.